### DISCUSSION TITLE
Arabisch-SchwiizDeutsch

### DISCUSSION THREAD

**Discussion by ahmadgneadiGoogle** [Estheryu991/SwissGerman_Dictionary#97] [Ideas]
Was ist deine Name warum bist du heute hier was ist deine Lieblingsessen was würdest du in fünf Jahren machen

**Comment by 0782264441** [comment]
ما اسمك، لماذا أنت هنا اليوم؟ ما هو طعامك المفضل؟ ماذا ستفعل بعد خمس سنوات؟
